src/components/contributors/contributorsList.tsx

```diff
@@ -1,6 +1,8 @@
-import React, { useState, useEffect } from "react";
```

Comment:
Removed unused imports, more extensive removal of redundant imports will be done later